### PR TITLE
depdelquery finish

### PR DESCRIPTION
**Pull request by Tail001**
1. depdelquery finish !
2. modify    ``` $env = parse_ini_file($_SERVER["DOCUMENT_ROOT"] . "\.env"); ``` in Dotenv.php file to original one

### CODE REVIEW

**Comment by dachangjames** [comment]
Why do you want to delete .env.sample????? Deleting it is not necessary at all.

**Comment by WaltsonZh** [comment]
> Why do you want to delete .env.sample????? Deleting it is not necessary at all.

In fact, Do not delete .env.sample

**Review by WaltsonZh — CHANGES_REQUESTED** [review]

File: dep/depdelquery.php [hunk 1]
```diff
@@ -1 +1,38 @@
...
+        <p>確認放棄查詢</p>
+    </div>
+    <?php
+        $del_table = DB::fetch_table("sl8gdm_stuapply_del");
+        function unit2Char($unit) {         
+            $unit_code = substr($unit, -2);
+            $unit_code = trim($unit_code) . "000";   
+            $query_set = DB::fetch_row("sl8gdm_dep","unit_parent", $unit_code);
+
+            return $query_set["unit_name"];
```
[the remark this answers is not in the document]
Comment (Tail001) [reply dep/depdelquery.php:17]: understand, i will  do it later

[the remark this answers is not in the document]
Comment (Tail001) [reply dep/depdelquery.php]: Recovery env.sample
unit2Char -> fetch DB ``` sl8gdm_permit_rec``` to get staff_cd , fetch ``` sl8gdm_dep``` gain unit_parent, and convert it to the Mandarin string

**Review by WaltsonZh — APPROVED** [review]
LGTM!!!!!!!!!!

**Pull request merged**